### PR TITLE
Upgrade to scalajs 1.7

### PR DESCRIPTION
**Pull request by clutroth**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -12,8 +12,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        scalaversion: ["2.11.12", "2.12.10", "2.13.1"]
-        scalajsversion: ["1.5.0"]
+        scalaversion: ["2.11.12", "2.12.15", "2.13.6"]
+        scalajsversion: ["1.7.0", "1.7.1"]
```
Comment (gzm0) [inline .github/workflows/ci.yml]: I do not quite understand how this is supposed to work: Won't we publish the CLI for a single Scala.js version only anyways? If so, what's the point of testing with an older one?

File: project/build.properties [hunk 2]
```diff
@@ -1 +1 @@
-sbt.version=1.2.8
+sbt.version=1.5.5
```
Comment (gzm0) [inline project/build.properties]: Put the sbt version upgrade into a separate commit to reduce noise?

**Review by gzm0 — CHANGES_REQUESTED** [review]
Only minor stuff.

**Comment by gzm0** [comment]
@clutroth thank you for your contributions to the CLI. I realize we have not even tagged a release after your last PR. Apologies for that.

@sjrd, I suggest we simply start releasing on its own versioning scheme? maybe with minor version bumps if we upgrade the minor Scala.js version? Definitely not the cleanest option, but better than not releasing at all :-/ WDYT?

**Comment by sjrd** [comment]
Yes, probably that's what we should do.

Ideally I think we should explore something using [`cs install`](https://get-coursier.io/docs/cli-install#application-descriptors) so that we only have to release to Maven Central, and not deal with the native runner scripts.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (clutroth) [reply .github/workflows/ci.yml]: You're right. It's pointless.

File: project/build.properties (hunk 2 above)
Comment (clutroth) [reply project/build.properties]: done

**Comment by clutroth** [comment]
Glad to help :) I'd appreciate release.

**Review by gzm0 — APPROVED** [review]

**Pull request merged**